src/tasks/FishingTask.py

```
@@ -0,0 +1,782 @@
+    def get_game_top_hwnd(self) -> int:
+        interaction = self.executor.interaction
+        hwnd_window = getattr(interaction, "hwnd_window", None)
+        if hwnd_window is not None:
+            return int(getattr(hwnd_window, "top_hwnd", 0) or getattr(hwnd_window, "hwnd", 0) or 0)
+        return int(getattr(interaction, "hwnd", 0) or 0)
+
+    def _ensure_foreground(self, hwnd: int):
+        try:
+            win32gui.ShowWindow(hwnd, win32con.SW_RESTORE)
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)